### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by rspt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
 <div class="ph4 ph5-m ph6-l">
   {% include components/header.html
       title=""
-      subtitle="In a constantly changing world, you need to focus on what’s important to you. That’s why at laruche we are specializing in improving your workflows using tomorrows tools to optimize your productive time."
+      subtitle="In a constantly changing world, you need to focus on what’s important to you. That’s why at laruche we are specializing in improving your workflows using tomorrow tools to optimize your productive time."
```
Comment (maxirozay) [inline index.html]: What do you mean by "productive time"? Productivity?

File: services/index.html [hunk 2]
```diff
@@ -16,14 +16,15 @@
     <h2 class="mt0">Consulting</h2>
     <p class="mb0 lh-copy">
       Regardless of the question we are here to answer it. From legacy systems
```
Comment (maxirozay) [inline services/index.html]: Maybe put a comma after "regardless of the question.

File: services/index.html [hunk 3]
```diff
@@ -35,7 +36,7 @@ <h2 class="mt0">Cloud Solutions</h2>
     <h2 class="mt0">Development</h2>
     <p class="mb0 lh-copy">
       Our web development expertise area range from static website to fully
```
Comment (maxirozay) [inline services/index.html]: Put a comma after static website?

Comment (maxirozay) [inline services/index.html]: You write "static website" and "web applications" (plural), so we should write "static websites".

File: services/index.html [hunk 4]
```diff
@@ -62,7 +63,7 @@ <h2 class="mt0">Audit</h2>
   <div class="w-100 w-50-l ml4-l pt4">
     <h2 class="mt0">IT Support</h2>
     <p class="mb0 lh-copy">
-      If you encounter some problems with you computers, security problems or
+      If you encounter some problems with your computers, security problems or
       general IT related questions we offer our expertize for your IT
```
Comment (maxirozay) [inline services/index.html]: Put a comma after related questions?

File: index.html (hunk 1 above)
Comment (rspt) [inline index.html]: No. The idea is that you need to be productive in your expertise zone, not outside of it spending time on IT/Web problems.

File: services/index.html (hunk 3 above)
Comment (rspt) [inline services/index.html]: No. "From to"

File: services/index.html (hunk 4 above)
Comment (rspt) [inline services/index.html]: Done

File: services/index.html (hunk 3 above)
Comment (rspt) [inline services/index.html]: Done (all in singular)

File: services/index.html (hunk 2 above)
Comment (rspt) [inline services/index.html]: Done

File: index.html (hunk 1 above)
Comment (maxirozay) [inline index.html]: Sounds weird for me, like it's a noun after a noun.

Comment (rspt) [inline index.html]: So how do you name the time when you are productive? ^^

Comment (maxirozay) [inline index.html]: Productivity time? I asked my girlfriend and if it's ok I merge.

Comment (rspt) [inline index.html]: Ok :)

Comment (maxirozay) [inline index.html]: So she says it's "your time productively".

Comment (rspt) [inline index.html]: Sorry but I doubt. I think we don't understand each other ^^

I mean by that in french: "C'est pourquoi chez laruche nous nous sommes spécialisés dans l'optimisation de vos processus à l'aide des outils de demain afin de maximiser votre temps productif". J'entends par "votre temps productif", le temps durant lequel tu exerces tes compétences et pas autre chose de moins important.

Comment (rspt) [inline index.html]: "your time productively" mean something different, like "votre temps productif". It should be "votre temps pendant lequel vous êtes productif"

**Comment by rspt** [comment]
Yes I wanted to use french, but I needed to change urls and page name and I'm lazy ^^

File: index.html (hunk 1 above)
Comment (rspt) [inline index.html]: Updated :)

**Pull request merged**